platform/platform-configuration-listener/src/main/java/org/codice/ddf/configuration/store/felix/PropertyConverter.java

```diff
@@ -104,4 +112,20 @@ public void accept(String line) {
     void setValueConverters(Map<String, PropertyValueConverter> valueConverters) {
         PropertyConverter.valueConverters = valueConverters;
     }
+
+    /**
+     * Trims and removes any brackets, parenthesis, and quotations from a value
```

Comment:
a little confused on the Javadoc here: "The cleaned string, or null is null is given". Looks like the "is" should be "if"